projects/igniteui-angular/src/lib/grids/grid/grid.component.ts

```diff
@@ -288,6 +294,12 @@ export class IgxGridComponent extends IgxGridBaseComponent implements IGridDataB
         }
     }
 
+    /**
+     *@hidden
+     */
+    @Output()
+    public groupingExpressionsChange = new EventEmitter<IGroupingExpression[]>();
```

Comment:
@MayaKirova It doesn't update the model if "Expand All/Collapse All" button is clicked from the UI.

Reply:
@mpavlinov It appears that groupingExpansionState is empty in both cases - when all are expanded or collapsed and the default state is controlled via an additional option (groupsExpanded) which is toggled on expand/collapse all. Hence the binding works, the values are perhaps misleading.